test/suites/storage_volume_initial_config.sh

```diff
@@ -0,0 +1,103 @@
+test_storage_volume_initial_config() {
+  pool="storage-initial-values"
+  profile="profile-initial-values"
+  image="testimage"
+
+  ensure_import_testimage
+
+  (
+    set -e
+
+    # Create storage pool and enable block_mode for new volumes.
+    lxc storage create "${pool}" zfs
+
+    # Create profile and set previously created storage pool as root disk device.
+    lxc profile create "${profile}"
+    lxc profile device add "${profile}" root disk path=/ pool="${pool}"
+
+
+    # > Verify zfs.block_mode without initial configuration.
+
+    # Verify "zfs.block_mode=true" is applied from pool configuration.
+    lxc storage set "${pool}" volume.zfs.block_mode=true
+
+    lxc launch "${image}" c --profile "${profile}"
```

Comment:
I'm trying to articulate what more is gained by using `launch` here rather than `init`. I can't really think of anything but maybe you had a specific reason?

Reply:
No, there is no particular reason. Will fix. Thanks :)